azure-run-tox-env.yml

```diff
@@ -32,5 +32,14 @@ jobs:
     - script: 'python -m tox -e coverage'
       displayName: generate coverage report
 
-    - script: 'python -m tox -e codecov -- -t $(CODECOV_TOKEN) --required -n "$(agent.os)-$(python.version)" --build "$(Build.DefinitionName)" --env OS=$(agent.os) pyt+hon=$(python.version)'
+    - script: |
+        for i in 1 2 3; do
```

Comment:
You can use `travis_retry` instead of the shell script

Reply:
Note there's no plan to use travis also. This is Azure DevOps, not Travis CI.